apps/app/src/views/routes/Order.tsx

```diff
@@ -135,6 +136,23 @@ const GET_ORDER = gql`
         carrier
         trackingNumber
       }
+      fulfillments {
```

Comment:
Now that we have this, can we get rid of (it would make the query faster):

```gql
fills {
        id
        prescription {
          id
          dispenseQuantity
          dispenseUnit
          fillsAllowed
          instructions
        }
        treatment {
          name
        }
        state
        requestedAt
        filledAt
      }
```

do we still need any of the fill information?